### PR TITLE
fix: Choosing Ticket holder same as ticket buyer not makes the text light gray 

### LINKED ISSUE #4004: Ticket buyer: Choosing Ticket holder same as ticket buyer makes the text light gray and hard to read
Ticket buyer: Choosing Ticket holder same as ticket buyer makes the text light gray and hard to read.

Screenshot from 2020-02-05 02-52-51

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #4004 
Screenshot from 2020-02-05 16-14-46
Screenshot from 2020-02-05 16-14-49

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
Screenshot from 2020-02-05 22-46-11

File: app/controllers/public/index.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default Controller.extend({
         order.set('attendees', attendees);
         await order.save()
           .then(order => {
-            this.notify.success(this.l10n.t('Order details saved. Please fill further details within 10 minutes.'));
+            this.notify.success(this.l10n.t('Order details saved. Please fill further details within 15 minutes.'));
```
Comment (kushthedude) [inline app/controllers/public/index.js]: Revert

**Comment by maze-runnar** [comment]
Actually this is a wrong msg, it should be 15 minutes , that's why i changed it to 15 in this pr ,as it is a very small change.

**Comment by kushthedude** [comment]
Even 15 minutes is wrong message.

**Comment by maze-runnar** [comment]
Should i change this to model.expiry-time something like this.

**Comment by kushthedude** [comment]
Yes

File: app/controllers/public/index.js [hunk 2]
```diff
@@ -131,7 +131,7 @@ export default Controller.extend({
         order.set('attendees', attendees);
         await order.save()
           .then(order => {
-            this.notify.success(this.l10n.t('Order details saved. Please fill further details within 10 minutes.'));
+            this.notify.success(this.l10n.t('Order details saved. Please fill further details within ' + this.settings.orderExpiryTime + ' minutes.'));
```
Comment (niranjan94) [inline app/controllers/public/index.js]: Codacy Issue found: [Unexpected string concatenation.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4948721)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4948721&bid=16356528)

**Pull request merged**